src/generator/writers/writeFunction.ts

```diff
@@ -465,6 +465,26 @@ export function writeStatement(
 
             return;
         }
+        case "statement_destruct": {
+            const t = getExpType(ctx.ctx, f.expression);
+            if (t.kind !== "ref") {
+                throwInternalCompilerError(
+                    `invalid destruct expression kind: ${t.kind}`,
+                    f.expression.loc,
+                );
+            }
+            const ty = getType(ctx.ctx, t.name);
+            const ids = ty.fields.map((field) => {
+                const id = Array.from(f.identifiers.entries()).find(
+                    ([key]) => key.text === field.name,
+                );
+                return id ? funcIdOf(id[1]) : "_";
```

Comment:
I'd say we should also translate Tact's `_` into the same underscore in FunC.

So, for instance, for the following snippet

```tact
struct S { x: Int; y: String; z: Bool }

let S { x: _, y } = ...;
```
we get underscores in the resulting FunC code both for `x` and `z`.

Reply:
Isn't it like this already?

Reply:
nope, for Tact's `_` you will get `$_`